src/ngraph/op/util/attr_types.hpp

```diff
@@ -368,6 +368,48 @@ namespace ngraph
         const DiscreteTypeInfo& get_type_info() const override { return type_info; }
     };
 
+    class NGRAPH_API DataHandle
```

Comment:
This provides access for a generic deserializer to initialize the data void* such as from a binary file. It provides the void* and it's length. Likewise, a deserializer has the void* and length of the data.

Reply:
I think I will get rid of DataHandle and just have a visitor method that gets passed the void* and size_t.

Reply:
Although the usage of `DataHandle` looks quite convenient it might be more explicit and readable that we're dealing with raw data if we just have another overload `on_attribute(const string& name, void* data_ptr, size_t size_in_bytes)` ?

Reply:
That's exactly what I just did.